jd4/compile.py

```diff
@@ -155,7 +155,7 @@ async def build(lang, code):
 def _init():
     try:
         with open(_LANGS_FILE) as file:
-            langs_config = yaml.load(file, Loader=yaml.RoundTripLoader)
+            langs_config = yaml.YAML().load(file)
```

Comment:
Constructing the YAML object looks expensive. Maybe create file-level globals.

Test with:

```
[yaml.YAML() for i in range(100000)][:0]

[y.YAML().load('a:1') for i in range(100000)][:0]

y = yaml.YAML()
[y.load('a:1') for i in range(100000)][:0]
```

Reply:
fixed